package.json

```
@@ -24,6 +24,7 @@
     "@badrap/valita": "0.1.3",
     "@marcj/marshal": "2.1.13",
     "@mojotech/json-type-validation": "3.1.0",
+    "@sinclair/typebox": "^0.24.3",
```

```suggestion
    "@sinclair/typebox": "0.24.3",
```

We use fixed versions, which get upgraded automatically by Renovate Bot.